features/expectation_framework_integration/aggregating_failures.feature

```diff
@@ -59,7 +59,8 @@ Feature: Aggregating Failures
                 # ./spec/use_block_form_spec.rb:18
                 # ./spec/use_block_form_spec.rb:10
 
-                1.1.1) Failure/Error: expect(response.status).to eq(200)
+                1.1.1) Failure/Error:
+                         expect(response.status).to eq(200)
```

Comment:
Oh you had already mentioned it :) https://github.com/rspec/rspec-core/pull/2083#issuecomment-146909531